drivers/nrf_wifi/fw_if/umac_if/src/fmac_api_common.c

```diff
@@ -901,12 +901,6 @@ int nrf_wifi_phy_rf_params_init(struct nrf_wifi_phy_rf_params *prf,
 	prf->pd_adjust_val.pd_adjt_hb_mid_chan = PD_ADJUST_VAL;
 	prf->pd_adjust_val.pd_adjt_hb_high_chan = PD_ADJUST_VAL;
 
-	/* Configure systematic offset value */
-	prf->syst_tx_pwr_offset.syst_off_lb_chan = SYSTEM_OFFSET_LB;
-	prf->syst_tx_pwr_offset.syst_off_hb_low_chan = SYSTEM_OFFSET_HB_CHAN_LOW;
-	prf->syst_tx_pwr_offset.syst_off_hb_mid_chan = SYSTEM_OFFSET_HB_CHAN_MID;
-	prf->syst_tx_pwr_offset.syst_off_hb_high_chan = SYSTEM_OFFSET_HB_CHAN_HIGH;
-
```

Comment:
these should be in a separate commit, as can be seen from commit log, we are clubbing multiple functionalities

Reply:
Handled in separate commit